docs/contributor/01-00-installation.md

```diff
@@ -90,50 +90,20 @@ kubectl delete -f config/samples/operator_v1alpha2_istio.yaml
 
 ## Install the Istio module with Lifecycle Manager locally on k3d
 
-To install the Istio module with Lifecycle Manager locally, apply the ModuleTemplate generated by the local build or use the artifacts that are created by `post-istio-module-build` job.
+To install the Istio module with Lifecycle Manager locally, use make or use the artifacts that are created by `post-istio-module-build` job.
 
 ### Install using the ModuleTemplate generated by the local build
 
-1. Set up a local k3d cluster and a local Docker registry.
+1. Provision k3d cluster, deploy Lifecycle-Manager, build the module and deploy it.
```

Comment:
```suggestion
1. Run the following command to provision a k3d cluster, deploy Lifecycle Manager, build the module, and deploy it.
```
The `make local-run` command is responsible for all those four actions, right?

Reply:
Yes